<?xml version="1.0" encoding="UTF-8"?>
<launch>

  
 
  
  <arg name="model_path" value="/3D-object-detection/openpcd_ros/src/deploy/pretrained_models/pv_rcnn_8369.pth"/>
  <arg name="cfg_path" value="/3D-object-detection/OpenPCDet/tools/cfgs/kitti_models/pv_rcnn.yaml"/>
  
  <arg name="input_pointcloud_topic" value="/os1_cloud_node/points_sync"/>
  
  <arg name="translate_x" value= "25"/>
  <arg name="threshold_score" value= "0.1"/>
   
 

  <!-- rosbag play -->
  <include file="$(find deploy)/launch/play_rosbag.launch" />

  

 
  <node pkg="deploy" type="openpcdet_node.py" name="openpcdet_node"  output="screen" required="true">
    <param name="model_path" value="$(arg model_path)"/>
    <param name="cfg_file" value="$(arg cfg_path)"/>
    <param name="input_pointcloud_topic" value="$(arg input_pointcloud_topic)"/>
    <param name="translate_x" value="$(arg translate_x)"/>
    <param name="threshold_score" value="$(arg threshold_score)"/>
    
  </node>

  
  

</launch>